src/NHibernate/Collection/Generic/SetHelpers/SetSnapShot.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-#if NETCOREAPP2_0
+#if NETCOREAPP2_0 || NETCOREAPP2_1
```

Comment:
This file is directly included as source in the test project too. So here we need to handle core 2.0 and 2.1 compilation.